cmd/fyne/internal/templates/data/hello_world_go

```diff
@@ -0,0 +1,14 @@
+package main
```

Comment:
I think the file extension for this should be `.got` to be consistent with other Go files in this directory.

Reply:
Haha, nice, thanks. Saw .got files but it didn't click... 😂